dlpy/tests/test_sequential.py

```diff
@@ -71,7 +71,7 @@ def test_2(self):
         Sequential(self.s, layers=layers, model_table='table2')
 
     def test_22(self):
```

Comment:
The reason I modified the test case is that Conv2d can only connect to a 3-d output. The output size of InputLayer without specifying width, height would be 0 and it raises an error when counting parameters.